cmake/modules/vvencInstall.cmake

```diff
@@ -111,7 +111,9 @@ function( resolve_target_interface_libs TGT OUT_VAR )
       resolve_target_interface_libs( ${lib} lib )
     endif()
 
-    list( APPEND ret ${lib} )
+    if( lib )
+      list( APPEND ret ${lib} )
+    endif()
```

Comment:
Should this change not be enough to fix the problem?

Reply:
Answered in a separate comment